### ISSUE TITLE
ShellCheck minimum version

### ISSUE THREAD

**Issue by Starfight** [sbaudoin/sonar-shellcheck#12]
I run sonar-scanner with ShellCheck from epel repository on CentOS7 and I have this warning:
```
WARN: Errors happened during analysis:
unrecognized option `-x'

Usage: shellcheck [OPTIONS...] FILES...
  -e CODE1,CODE2..  --exclude=CODE1,CODE2..  exclude types of warnings
  -f FORMAT         --format=FORMAT          output format
  -s SHELLNAME      --shell=SHELLNAME        Specify dialect (bash,sh,ksh)
  -V                --version                Print version information
```

It seems that version available with epel (0.3.8) not working with sonar-shellcheck. But tested with latest version of shellcheck (0.6.0) and option `-x` is available. Can you add the minimum version needed for ShellCheck in documentation?
Thanks for your work.

**Comment by oliyavan** [comment]
> I run sonar-scanner with ShellCheck from epel repository on CentOS7 and I have this warning:
> 
> ```
> WARN: Errors happened during analysis:
> unrecognized option `-x'
> 
> Usage: shellcheck [OPTIONS...] FILES...
>   -e CODE1,CODE2..  --exclude=CODE1,CODE2..  exclude types of warnings
>   -f FORMAT         --format=FORMAT          output format
>   -s SHELLNAME      --shell=SHELLNAME        Specify dialect (bash,sh,ksh)
>   -V                --version                Print version information
> ```
> 
> It seems that version available with epel (0.3.8) not working with sonar-shellcheck. But tested with latest version of shellcheck (0.6.0) and option `-x` is available. Can you add the minimum version needed for ShellCheck in documentation?
> Thanks for your work.

Thanks for the update, i also faced the same issue in ubuntu16.04 box, but the default ubuntu repo also dont have the  minimum 0.4.0 version it only have 0.3.8 

i downloaded and copied the binary manually, now it is working

Thanks for your ticket @Starfight , based on your ticket only i find the soulution
